File: Guide.cs
class Guide
{
    public static void ShowGuide()
    {
        Console.WriteLine("Dream Journal CLI\n");
        Console.WriteLine("=================\n\n");

        Console.WriteLine("USAGE:\n");
        Console.WriteLine("  command [options] [arguments]\n\n");

        Console.WriteLine("COMMANDS:\n");

        Console.WriteLine("  add <title> <content> <lucid>\n");
        Console.WriteLine("    Add a new dream entry to the journal.\n");
        Console.WriteLine("    - title   : String (use quotes if it contains spaces)\n");
        Console.WriteLine("    - content : String (use quotes if it contains spaces)\n");
        Console.WriteLine("    - lucid   : Boolean (true or false)\n");
        Console.WriteLine("    Example:\n");
        Console.WriteLine("      add \"Flying Dream\" \"I was flying over a city\" true\n\n");

        Console.WriteLine("  remove <title>\n");
        Console.WriteLine("    Remove a dream entry by its title.\n");
        Console.WriteLine("    Example:\n");
        Console.WriteLine("      remove \"Flying Dream\"\n\n");

        Console.WriteLine("  journal\n");
        Console.WriteLine("    Display all journal entries.\n\n");

        Console.WriteLine("  journal -limit <number>\n");
        Console.WriteLine("    Display the last <number> of entries.\n");
        Console.WriteLine("    Example:\n");
        Console.WriteLine("      journal -limit 5\n\n");

        Console.WriteLine("  change -path <path>\n");
        Console.WriteLine("    Set the folder where the journal is saved.\n");
        Console.WriteLine("    Example:\n");
        Console.WriteLine("      change -path C:\\\\DreamJournal\n\n");

        Console.WriteLine("  change -startup-launch <Y/N>\n");
        Console.WriteLine("    Enable or disable launch on Windows startup.\n");
        Console.WriteLine("    Example:\n");
        Console.WriteLine("      change -startup-launch Y\n\n");

        Console.WriteLine("  donate\n");
        Console.WriteLine("    Open the Ko-fi donation page in your browser.\n");
    }
}

File: guide.c
#include <stdio.h>

void ShowGuide()
{
    printf("    ____  ____  _____    _    __  __            \n");
    printf("    |  _ \\|  _ \\| ____|  / \\  |  \\/  |           \n");
    printf("    | | | | |_) |  _|   / _ \\ | |\\/| |           \n");
    printf("    | |_| |  _ <| |___ / ___ \\| |  | |           \n");
    printf("    |____/|_|_\\_\\_____/_/__ \\_\\_|_ |_|_    _     \n");
    printf("        | |/ _ \\| | | |  _ \\| \\ | |  / \\  | |    \n");
    printf("    _  | | | | | | | | |_) |  \\| | / _ \\ | |    \n");
    printf("    | |_| | |_| | |_| |  _ <| |\\  |/ ___ \\| |___ \n");
    printf("    \\___/ \\___/ \\___/|_| \\_\\_| \\_/_/   \\_\\_____|  \n");

    printf("USAGE:\n");
    printf("  command [options] [arguments]\n\n");

    printf("COMMANDS:\n");

    printf("  add <title> <content> <lucid>\n");
    printf("    Add a new dream entry to the journal.\n");
    printf("    - title   : String (use quotes if it contains spaces)\n");
    printf("    - content : String (use quotes if it contains spaces)\n");
    printf("    - lucid   : Boolean (true or false)\n");
    printf("    Example:\n");
    printf("      add \"Flying Dream\" \"I was flying over a city\" true\n\n");

    printf("  remove <title>\n");
    printf("    Remove a dream entry by its title.\n");
    printf("    Example:\n");
    printf("      remove \"Flying Dream\"\n\n");

    printf("  journal\n");
    printf("    Display all journal entries.\n\n");

    printf("  journal -limit <number>\n");
    printf("    Display the last <number> of entries.\n");
    printf("    Example:\n");
    printf("      journal -limit 5\n\n");

    printf("  change -path <path>\n");
    printf("    Set the folder where the journal is saved.\n");
    printf("    Example:\n");
    printf("      change -path C:\\\\DreamJournal\n\n");

    printf("  change -startup-launch <Y/N>\n");
    printf("    Enable or disable launch on Windows startup.\n");
    printf("    Example:\n");
    printf("      change -startup-launch Y\n\n");

    printf("  donate\n");
    printf("    Open the Ko-fi donation page in your browser.\n");
}